src/allmydata/test/test_testing.py

```diff
@@ -0,0 +1,38 @@
+from twisted.trial.unittest import (
+    TestCase,
+)
+from twisted.internet.defer import (
+    inlineCallbacks,
+)
+
+from allmydata.testing.web import (
+    create_tahoe_treq_client,
+)
+
+
+class FakeWebTest(TestCase):
+    """
+    Test the WebUI verified-fakes infrastucture
+    """
+
+    def setUp(self):
+        super(FakeWebTest, self).setUp()
+        self.http_client = create_tahoe_treq_client()
+
+    @inlineCallbacks
+    def test_create_and_download(self):
+        """
+        Upload some content and then download it
+        """
+        content = "fake data\n" * 100
+        resp = yield self.http_client.put("http://example.com/uri", content)
+        cap = yield resp.content()
```

Comment:
Shall we assert something about the response code as well?  Is it CREATED or just OK?

Reply:
Hmm, `tahoe put` accepts either 200 or 201 .. but from `filenode.py` it looks like it should be 201 (because it's a brand-new file; other cases return 200).